python/lsst/ip/isr/isrTask.py

```diff
@@ -859,9 +857,21 @@ class IsrTaskConfig(pipeBase.PipelineTaskConfig,
     # Vignette correction configuration.
     doVignette = pexConfig.Field(
         dtype=bool,
-        doc="Apply vignetting parameters?",
+        doc="Compute and attach validPolygon to exposure according to vignetting parameters?",
         default=False,
     )
+    doMaskVignettePolygon = pexConfig.Field(
+        dtype=bool,
+        doc=("Add a mask bit for pixels within the vingetted region?  Ignored if doVignette "
+             "is False"),
+        default=True,
+    )
+    vignetteValue = pexConfig.Field(
+        dtype=float,
+        doc="Value to replance image array pixels with in the vingetted region?  Ignored if None.",
```

Comment:
Typo: "vingetted"

Reply:
Totally inherited from me...sorry!  And I'm also questioning my decision to add a question mark here!